assignments/game-of-life/code.js

```diff
@@ -0,0 +1,54 @@
+drawFilledRect(0, 0, width, height, 'black')
+
+
+
+const makeArrayRow = () => {
+  let row = []
+  for (let i = 0; i < width; i += 5) {
+    if (Math.random() < .2) {
+      row.push(1)
+    } else {
+      row.push(0)
+    }
+  }
+  return row
+}
```

Comment:
I was going to leave a comment about how this should also probably use a loop with `i < width / 5` and then a regular `i++` but it occurs to me that even better would be to make a change that's in the same spirit as some of the changes we made today in class. This function obviously depends on a global variable `width` but there's no real reason for it to. The width is just giving us the number of elements to push into each row so why not make that an argument to this function:

```suggestion
const makeArrayRow = (columns) => {
  let row = []
  for (let i = 0; i < columns; i++) {
    if (Math.random() < .2) {
      row.push(1)
    } else {
      row.push(0)
    }
  }
  return row
}
```

That makes this function easier to understand because we don't have to know about anything outside of the function itself. Obviously, making this change is going to require some changes at the place we call this but I'll address that in another comment.